WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift

```diff
@@ -1,5 +1,19 @@
 extension FancyAlertViewController {
     private struct Strings {
+        static let titleText = NSLocalizedString("Want a little help getting started?", comment: "Title of alert asking if users want to try out the quick start checklist.")
+        static let bodyText = NSLocalizedString("We’ll walk you through the basics of building and growing your site", comment: "Body text of alert asking if users want to try out the quick start checklist.")
+        static let allowButtonText = NSLocalizedString("Yes, help me", comment: "Allow button title shown in alert asking if users want to try out the quick start checklist.")
```

Comment:
The capitalization of the allow and not not buttons should probably be the same? (`Yes, help me` and `Not This Time`)